src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Windows.cs

```diff
@@ -329,10 +329,9 @@ public static int GetProcessIdFromHandle(SafeProcessHandle processHandle)
 
         public static ProcessInfo[] GetProcessInfos(string machineName, bool isRemoteMachine)
         {
-            PerformanceCounterLib? library = null;
             try
             {
-                library = PerformanceCounterLib.GetPerformanceCounterLib(machineName, new CultureInfo("en"));
+                PerformanceCounterLib library = PerformanceCounterLib.GetPerformanceCounterLib(machineName, new CultureInfo("en"));
```

Comment:
The comment below is now stale; at least previously library was in scope there... now it's not.